scrapi/harvesters/rcaap.py

```diff
@@ -0,0 +1,19 @@
+'''
+Harvester for the RCAAP - Repositório Científico de Acesso Aberto de Portugal for the SHARE project
+
+Example API call: http://www.rcaap.pt/oai?verb=ListRecords&metadataPrefix=oai_dc
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class RcaapHarvester(OAIHarvester):
+    short_name = 'rcaap'
+    long_name = 'RCAAP - Repositório Científico de Acesso Aberto de Portugal'
```

Comment:
I think this file is interpreted as `latin-1` by default in python2, so you'll need to change this character or declare the encoding of the file to be `utf-8`

https://www.python.org/dev/peps/pep-0263/